test/test_integration.py

```diff
@@ -245,6 +312,112 @@ def test_vertical_slider(self):
         for entry in self.get_log():
             raise Exception('browser error logged during test', entry)
 
+    def test_loading_range_slider(self):
```

Comment:
For both slider and range slider, make sure that the horizontal / vertical version displays correctly using the previously used styling, that the slider nubs can be clicked without error and that loading state is applied correctly to the DOM element